lagom-cloud-object-storage-example/account-impl/src/main/java/com/lightbend/lagom/account/impl/readside/AccountBalanceReportProcessor.java

```diff
@@ -0,0 +1,63 @@
+package com.lightbend.lagom.account.impl.readside;
+
+import akka.Done;
+import akka.japi.Pair;
+import akka.stream.javadsl.Flow;
+import com.lightbend.lagom.account.impl.AccountEvent;
+import com.lightbend.lagom.javadsl.persistence.AggregateEventTag;
+import com.lightbend.lagom.javadsl.persistence.Offset;
+import com.lightbend.lagom.javadsl.persistence.ReadSideProcessor;
+import org.pcollections.PSequence;
+import org.pcollections.TreePVector;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class AccountBalanceReportProcessor extends ReadSideProcessor<AccountEvent> {
+
+  final private AccountReportRepository repository;
+
+  @Inject
+  public AccountBalanceReportProcessor(AccountReportRepository repository) {
+    this.repository = repository;
+  }
+
+
+  @Override
+  public PSequence<AggregateEventTag<AccountEvent>> aggregateTags() {
+    return TreePVector.singleton(AccountEvent.TAG);
```

Comment:
That's great! I am using sharding on the demo lagom-eventstore so we¡k have both examples covered. 👍

Reply:
Hmm, after reading the repository it'd be good if we add a comment here linking to the sharding docs and briefly detail the implications of not sharding. WDYT?

Reply:
I was not planing to for further than that since this is not in the scope of the demo.